client/lib/wp/sync-handler/cache-invalidation.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * External dependencies
+ */
+import moment from 'moment';
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import warn from 'lib/warn';
+import { getLocalForage } from 'lib/localforage';
+
+/**
+ * Module variables
+ */
+const localforage = getLocalForage();
+const debug = debugFactory( 'calypso:sync-handler:cache' );
+const RECORDS_LIST_KEY = 'records-list';
+// set record lifetime to 2 days
+const LIFETIME = 1000 * 60 * 60 * 24 * 2;
```

Comment:
From: https://github.com/Automattic/wp-calypso/pull/3271#discussion_r52773960 I think we have a preference for something like the following:
```javascript
const DAY_IN_HOURS = 24;
const HOUR_IN_MS = 3600000;
export const LIFETIME = 2 * DAY_IN_HOURS * HOUR_IN_MS;
```